extensions/pyRevitTools.extension/checks/cad_audit_check.py

```diff
@@ -0,0 +1,189 @@
+            radio_text (``str``): RadioButton label Text
+            default (``bool``): Sets IsChecked state [Default: False]
+            wpf_params (kwargs): Additional WPF attributes
+        """
+        self.Name = name
+        self.Content = radio_text
+        self.IsChecked = default
+        self.set_attrs(top_offset=0, **kwargs)
+
+    @property
+    def value(self):
+        return self.IsChecked
+    
+def get_cad_site(cad_inst):
+    """ A CAD's location site cannot be got from the Shared Site parameter 
+        cad_inst.Name returns the site name with a 'location' prefix (language-specific, eg 'emplacement' in French)"""
+    cad_site_name = cad_inst.Name
+    cad_site_name.replace("location", "-")
+    return cad_site_name
+
+
+def get_user_input():
+    """ create RPW input FlexForm for user choice of collection mode (coll_mode) whole model or just active view """
+    flexform_comp = [Label("Audit CAD instances:")]
+    flexform_comp.append(RadioButton("model", "in this project", True, GroupName="grp")) # GroupName implemented in class through kwargs
+    flexform_comp.append(RadioButton("active_view", "in only the active view", False, GroupName="grp"))
+    flexform_comp.append(Separator())
+    flexform_comp.append(Button("CANCEL", on_click=ButtonClass.cancel_clicked))
+    flexform_comp.append(Button("OK"))
+
+    user_input = FlexForm("Audit of CAD instances in the current project", flexform_comp, Width=500, Height=200) # returns a FlexForm object
+    user_input.show()
+    user_input_dict = user_input.values # This is a dictionary
+    if not bool(user_input_dict): # check if dict not empty (a bool of non-empty dict is True)
+        script.exit()
+
+    return user_input_dict
+
+def get_load_stat(cad, is_link):
+    """ Loaded status from the import instance's CADLinkType """
+    cad_type = doc.GetElement(cad.GetTypeId()) # Retreive the type from the instance
+    
+    if is_link:
+        exfs = cad_type.GetExternalFileReference()
+        status = exfs.GetLinkedFileStatus().ToString()
+        if status == "NotFound":
+            status = ":cross_mark: NotFound"
+        elif status == "Unloaded":
+            status = ":heavy_multiplication_x: Unloaded"        
+    else:
+        status = ":warning: IMPORTED" # Not an external reference
+
+    return status
+
+
+def checkModel(doc, output):
+    timer = Timer()
+    output = script.get_output()
+    output.close_others()
+    output.set_title("CAD audit of model '{}'".format(doc.Title))
+    output.set_width (1700)
+    
+    coll_mode = get_user_input()["active_view"]
+    
+    table_data = [] # store array for table formatted output
+    row_head = ["No", "Select/Zoom", "DWG instance", "Loaded status", "Workplane or single view", "Duplicate", "Workset", "Creator user", "Location site name"] # output table first and last row
+    cad_instances = collect_cadinstances(coll_mode)
+    for cad in cad_instances:
+        count = 0
```

Comment:
count is always 0 here...
```suggestion
    for count, cad in enumerate(cad_instances, start=1):
```